usaspending_api/search/v2/urls.py

```diff
@@ -4,13 +4,17 @@
 from usaspending_api.search.v2.views.spending_by_award import SpendingByAwardVisualizationViewSet
 from usaspending_api.search.v2.views.spending_by_award_count import SpendingByAwardCountVisualizationViewSet
 from usaspending_api.search.v2.views.spending_by_category import SpendingByCategoryVisualizationViewSet
+from usaspending_api.search.v2.views.spending_by_category_views.awarding_agency import AwardingAgencyViewSet
+from usaspending_api.search.v2.views.spending_by_category_views.awarding_subagency import AwardingSubagencyViewSet
 from usaspending_api.search.v2.views.spending_by_geography import SpendingByGeographyVisualizationViewSet
 from usaspending_api.search.v2.views.spending_over_time import SpendingOverTimeVisualizationViewSet
 
 urlpatterns = [
     url(r"^new_awards_over_time", NewAwardsOverTimeVisualizationViewSet.as_view()),
     url(r"^spending_by_award_count", SpendingByAwardCountVisualizationViewSet.as_view()),
     url(r"^spending_by_award", SpendingByAwardVisualizationViewSet.as_view()),
+    url(r"^spending_by_category/awarding_agency", AwardingAgencyViewSet.as_view()),
```

Comment:
Should  the new spending_by_category endpoints be in their own url file? There will be over a dozen by the time all of the endpoints are created